nuntium/models.py

```diff
@@ -168,7 +169,7 @@ def recently_confirmated(self):
 
     @property
     def people(self):
-        people = Person.objects.filter(
+        people = PopoloPerson.objects.filter(
```

Comment:
Same with this line, I think this should probably only be changed once the data has been migrated in later commits?